src/Grpc.Net.ClientFactory/Internal/GrpcCallInvokerFactory.cs

```diff
@@ -190,4 +192,15 @@ public override void SetCompositeCredentials(object state, ChannelCredentials ch
         {
         }
     }
+
+    private static class Log
+    {
+        private static readonly Action<ILogger, string, Exception?> _unsupportedHttpClientActions =
+            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(1, "UnsupportedHttpClientActions"), "The ConfigureHttpClient method is not supported when creating gRPC clients. Unable to apply configuration to client with the name '{ClientName}'.");
```

Comment:
This appears to indicate a misconfiguration - maybe it should be a warning?

Reply:
I've changed the log message content. I think debug is the right level for the new message.